src/main/modOrganizer.ts

```diff
@@ -2,11 +2,34 @@ import path from "path";
 import childProcess from "child_process";
 import { USER_PREFERENCE_KEYS, userPreferences } from "@/main/config";
 import { logger } from "@/main/logger";
+import { copyGameFiles, checkGameFiles } from "@/main/gameFiles";
+import { copyEnbFiles, checkEnbFiles } from "@/main/enb";
 
 export const MO2EXE = "ModOrganizer.exe";
 
-export function launchGame() {
+export async function launchGame() {
   try {
+    const gameFilesExist = await checkGameFiles(
+      userPreferences.get(USER_PREFERENCE_KEYS.MOD_DIRECTORY),
+      userPreferences.get(USER_PREFERENCE_KEYS.SKYRIM_DIRECTORY)
+    );
+    logger.info(gameFilesExist);
```

Comment:
We should probably add some context to this.
Also, probably more of a debug log
```suggestion
    logger.debug(`Game files exist on launch: ${gameFilesExist}`);
```